### PR TITLE
[Merged by Bors] - feat: complex differentiable functions are continuously differentiable

### PR DESCRIPTION
**Pull request by ocfnash**
---

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

File: Mathlib/Analysis/Analytic/Basic.lean [hunk 1]
```diff
@@ -1379,6 +1379,16 @@ theorem isOpen_analyticAt : IsOpen { x | AnalyticAt 𝕜 f x } := by
   exact mem_of_superset (EMetric.ball_mem_nhds _ hr.r_pos) fun y hy => hr.analyticAt_of_mem hy
 #align is_open_analytic_at isOpen_analyticAt
 
+variable {𝕜}
+
+theorem AnalyticAt.eventuallyAnalyticAt {f : E → F} {x : E} (h : AnalyticAt 𝕜 f x) :
+    ∀ᶠ y in 𝓝 x, AnalyticAt 𝕜 f y :=
+(isOpen_analyticAt 𝕜 f).mem_nhds h
+
+theorem AnalyticAt.existsMemNhdsAnalyticOn {f : E → F} {x : E} (h : AnalyticAt 𝕜 f x) :
```
Comment (sgouezel) [inline Mathlib/Analysis/Analytic/Basic.lean:1388]: I think the naming convention is wrong. I would say `AnalyticAt.exists_mem_nhds_analyticOn`, but I'm not sure. The way I understand it is: separate different mathematical concepts with underscores, and start each mathematical concept with a lowercase letter.

You should ask on zulip, then we would both learn the right answer :-)

(Same thing for the previous lemma, my guess is that it should be `AnalyticAt.eventually_analyticAt`)

File: Mathlib/Analysis/Complex/CauchyIntegral.lean [hunk 2]
```diff
@@ -611,6 +612,11 @@ protected theorem _root_.Differentiable.analyticAt {f : ℂ → E} (hf : Differe
   hf.differentiableOn.analyticAt univ_mem
 #align differentiable.analytic_at Differentiable.analyticAt
 
+/-- A complex differentiable function `f : ℂ → E` is continuously differentiable at every point. -/
+protected theorem _root_.Differentiable.contDiff {f : ℂ → E} (hf : Differentiable ℂ f) {n : ℕ∞} :
+    ContDiff ℂ n f :=
+  contDiff_iff_contDiffAt.mpr $ fun z ↦ (hf.analyticAt z).contDiffAt
```
Comment (urkud) [inline Mathlib/Analysis/Complex/CauchyIntegral.lean:618]: Could you please add a `DifferentiableOn` version (for an open set)?

Comment (ocfnash) [reply Mathlib/Analysis/Complex/CauchyIntegral.lean:624]: Good point, now done!

**Comment by sgouezel** [comment]
bors r+
Thanks!

**Comment by ocfnash** [comment]
For some reason bors has ignored the instruction above. Let's try again:

bors merge

**Pull request closed without merging**